packages/extensionmanager-extension/src/index.ts

```diff
@@ -57,6 +57,7 @@ const plugin: JupyterFrontEndPlugin<void> = {
       v.id = 'extensionmanager.main-view';
       v.title.icon = extensionIcon;
       v.title.caption = trans.__('Extension Manager');
+      v.node.setAttribute('data-testid', 'extension-manager-section');
```

Comment:
What is `data-testid`?

Reply:
It's used by getByTestId in Galata to get the extension manager section. I tried using page.locator() with the id 'extensionmanager.main-view' but that returned multiple elements.

Reply:
In the commit history I see that you tried `getByTitle` but I do not see you using the ID locator directly (instead of using `getByTitle`. I think this is not needed - can you give using just ID another try please?

Reply:
Hey @krassowski, I've tried just using the ID locator and that can't find the element. If I try and click on the extension manager it returns two elements. I've tried first() and last() but those time out too. I think it's because there are two elements that have an id or data-id of extensionmanager.main-view. Adding the data-testid was the solution that I found that worked.

Reply:
Do you see two elements with `extensionmanager.main-view` ID in the page? That would be a pretty serious bug. My guess is that there was some mistake in the selector, for example unescaped dot `.`. Did you try `#extensionmanager\\.main-view`?

Reply:
The extension manager section has an ID of extensionmanager.main-view. The extension manager li has a data-id of extensionmanager.main-view. 

I'll try it with an escaped character. Thanks, I didn't think of that.

Reply:
The test now passes after adding an escaped character.